pkg/metrics/metrics.go

```diff
@@ -28,17 +28,19 @@ var (
 
 // gauges
 var (
-	// DEPRECATED - See UserAccountGaugeVec
+	// DEPRECATED - See UserAccountGaugeVec and MasterUserRecordGaugeVec
```

Comment:
```suggestion
	// DEPRECATED - See MasterUserRecordGaugeVec
```
I guess the reference to `UserAccountGaugeVec` was when we had a `UserAccountGauge` in the past?

Reply:
right, that's probably a quick copy/paste error of mine